src/components/analyses/details/InfoPanel.js

```diff
@@ -162,7 +94,11 @@ function Step(props) {
                         btnText={t("copyAnalysisId")}
                     />
                 </div>
-                <Updates updates={updates} baseId={baseId} t={t} />
+                <Updates
+                    updates={updates}
+                    baseId={build(baseId, ids.INFO.UPDATE)}
+                    t={t}
```

Comment:
Looks like this `t` prop is passed to `Updates` but that component is not using it